src/sentry/monitors/models.py

```diff
@@ -305,11 +305,10 @@ def get_next_expected_checkin_latest(self, last_checkin: datetime) -> datetime:
         most recent checkin time. This is determined by the user-configured
         margin.
         """
+        from sentry.monitors.utils import get_checkin_margin_timedelta
+
         next_checkin = self.get_next_expected_checkin(last_checkin)
-        # TODO(epurkhiser): We should probably just set this value as a
-        # `default` in the validator for the config instead of having the magic
-        # default number here
```

Comment:
I think this comment is still somewhat valid

Reply:
sure. I can movie it